### PR TITLE
Build a Product Landing Page

### PR DESCRIPTION
**Pull request by Chandan-devs-tech**
## Landing Page

# In index.html I have structured a website :
- With a logo
- Using ul tag to list items.
- Added an Input box and a button to it.
# In style.css I have styled a website:
-With positioning elements.
-Adding a hover effect to the button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    
+          <div class="container"></div>
+    
+          <section id="hero">
+            <h2>Handcrafted, home-made masterpieces</h2>
+            <form id="form" action="https://www.freecodecamp.com/email-submit">
+              <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
+              <input id="submit" type="submit" value="Get Started" class="btn">
+            </form>
+          </section>
```
Comment (Chandan-devs-tech) [inline index.html:34]: - Please add a label tag. Label tag make the site more readable and accessible for screen readers.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        padding: 0px 20px;
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+        background-color: #eee;
+      } 
+      input[type="submit"]:hover {
+        background-color: orange;
+        transition: background-color 1s;
+      }
```
Comment (Chandan-devs-tech) [inline style.css:25]: - Please make an Indentation of two spaces. This will help your code to be clean.

**Review by Chandan-devs-tech — COMMENTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 
Good job so far!
- There are some issues that you still need to work on to go to the next project but you are almost there!
- You have done really impressive work. You took care of even minute things. Keep up the good work
  - Please check the comment below and make changes to it.

**Review by bienvenuushindi — APPROVED** [review]
Hi @Chandan-devs-tech    , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights
- Used two comments ✔️ 
- Comments are clear and concise ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**